etc/behat/services/contexts/ui.xml

```diff
@@ -14,6 +14,7 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
         <parameter key="sylius.behat.context.ui.admin.managing_countries.class">Sylius\Behat\Context\Ui\Admin\ManagingCountriesContext</parameter>
+        <parameter key="sylius.behat.context.ui.admin.managing_currency.class">Sylius\Behat\Context\Ui\Admin\ManagingCurrenciesContext</parameter>
```

Comment:
It should be named ``sylius.behat.context.ui.admin.managing_currencies.class`` and similarly service using this parameter.